packages/iris-grid/src/sidebar/conditional-formatting/ConditionalFormatEditor.tsx

```diff
@@ -17,16 +17,21 @@ const log = Log.module('ConditionalFormatEditor');
 
 export type SaveCallback = (rule: FormattingRule) => void;
 
+export type UpdateCallback = (rule?: FormattingRule) => void;
+
 export type CancelCallback = () => void;
 
-export type ChangeCallback = (ruleConfig: BaseFormatConfig) => void;
+export type ChangeCallback = (
```

Comment:
I don't like this import cycle... ConditionalFormatEditor imports RowFormatEditor and ColumnFormatEditor, but they both import `ChangeCallback` from `ConditionalFormatEditor`. Put `ChangeCallback` into it's own file that's only imported by RowFormatEditor/ColumnFormatEditor (Or ConditionalFormattingUtils?)

Reply:
Moved to `ConditionalFormattingUtils`